### DISCUSSION TITLE
mrp

### DISCUSSION THREAD

**Discussion by karaportti** [blueseerERP/blueseer#33] [Q&A] [answered]
Hi, I am wondering abou the MRP. When I run the MRP all the demand based on BOM is on the same day (meaning that on that day there is inventory shortrage) eventhough I have updated leadtimes. With other ERP systems the MRP demands are based o the leadtime on item master file. 

Have I done something wrong? Is there also a parameter that leadtimes are days, weeks or even hours?

**Comment by vaughnte** [comment, marked as answer]
Hello Karaportti,

You are correct.....the MRP demand date for components falls on the demand date of the parent....in other words, there is no lead-time baked into the calculation.   The current MRP Browse menu has 'actual' demand, purchase, QOH dates for the components to make it easier to visualize the change in the browse menu.   Right now, the lead-time field in the item master is currently 'informational only'.   I am considering adding an adjustment in the next revision that gives the enduser the option to operate the lead-time field as either 'informational only' or as a live field applied to calculating the required date....i.e.... component demand date  = (parent demand date - parent lead time (days)).

**Reply by vaughnte** [reply]
Also...the leadtime field is considered 'days'.   There is no parameter unit of measure for this field at this time.
